src/core/qgsvectorfilewriter.cpp

```diff
@@ -3172,6 +3187,10 @@ QList< QgsVectorFileWriter::FilterFormatDetails > QgsVectorFileWriter::supported
         return true; // Make https://twitter.com/shapefiIe a sad little fellow
       else if ( b.driverName == QLatin1String( "GPKG" ) )
         return false;
+      else if ( a.driverName == QLatin1String( "FlatGeobuf" ) )
```

Comment:
Let's drop these sorting tweaks for now and revisit when we have a better idea of the format's adoption -- I think it's premature to push users to this format at this stage.

Reply:
Agreed :)

Reply:
And done.